go.mod

```diff
@@ -1,6 +1,8 @@
 module istio.io/istio
 
-go 1.22.0
+go 1.23.0
+
+toolchain go1.23.7
```

Comment:
@istio/release-managers-1-24 @howardjohn is this necessary/sufficient for our build process or would we want changes over in https://github.com/istio/tools/tree/release-1.24?

Reply:
So 1.24 is already on Go 1.24. This just bumps the minimum. This _also_ unlocks some behavioral changes; go does a thing similar to istio compatibility versions, so before we were "Go 1.23 with compatVersion=1.22".

Now we would be bumping that.

This differs from 1.23, which was entirely on Go 1.22.